<launch>
    <arg name="camera_name" default="yeet_camera1"/>
    <arg name="host" default="0.0.0.0"/>
    <arg name="model_name" default="faster-rcnn"/>
    <arg name="score_thresh" default="0.5"/>

    <arg name="point_selection_method" default="std"/>
    <arg name="point_selection_distance_thresh" default="1.0"/>
    <arg name="point_selection_method_thresh" default="0.5"/>
    <arg name="plot_picking_points" default="True"/>

    <node pkg="rasberry_perception" name="rasberry_perception_detector" type="detector_2d.py" output="screen">
        <param name="camera_name" type="str" value="$(arg camera_name)"/>
        <param name="host" type="str" value="$(arg host)"/>
        <param name="model_name" type="str" value="$(arg model_name)"/>
        <param name="score_thresh" type="double" value="$(arg score_thresh)"/>
        <param name="point_selection_method" value="$(arg point_selection_method)"/>
        <param name="point_selection_distance_thresh" value="$(arg point_selection_distance_thresh)"/>
        <param name="point_selection_method_thresh" value="$(arg point_selection_method_thresh)"/>
        <param name="plot_picking_points" value="$(arg plot_picking_points)"/>
    </node>
</launch>
